LLama/LLamaContext.cs

```diff
@@ -153,6 +154,8 @@ public string DeTokenize(IReadOnlyList<LLamaToken> tokens)
         }
 
         #region state load/save
+        private const uint FileHeaderMagic = 3430400180;
```

Comment:
It's there as a sanity check - this is the first 4 bytes of any compatible file and if you try to decode an incompatible file it fails fast because this is the very first thing checked. The actual value is totally arbitrary.